charts/consent/templates/deployment.yaml

```
@@ -0,0 +1,155 @@
+          {{- end }}"
+          imagePullPolicy: Always
+          ports:
+            - containerPort: 8080
+            - name: metrics
+              containerPort: 9090
+              protocol: TCP
+          securityContext:
+            runAsUser: 2  # non-root user
+            allowPrivilegeEscalation: false
+          env:
+            - name: SENTRY_DSN
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Chart.Name }}-secrets
+                  key: dsn
+            - name: PROMETHEUS_ARGS
+              value: "-javaagent:/etc/prometheusjmx/prometheusjmx.jar=9090:/etc/consent-cm/prometheusJmx.yaml"
+            - name: DB_USER
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Chart.Name }}-secrets
+                  key: databaseUser
+            - name: DB_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Chart.Name }}-secrets
+                  key: databasePassword
+            - name: SENDGRID_API
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Chart.Name }}-secrets
+                  key: sendgridApiKey
+          command: ["java"]
+          args:
+          - -Dsentry.stacktrace.app.packages=org.broadinstitute
+          - -Dsentry.dsn=$(SENTRY_DSN)
+          - -Ddw.database.user=$(DB_USER)
+          - -Ddw.database.password=$(DB_PASSWORD)
+          - -Ddw.mailConfiguration.sendGridApiKey=$(SENDGRID_API)
+          - $(PROMETHEUS_ARGS)
+          - -jar
+          - /opt/consent.jar
+          - server
+          - /etc/consent-cm/consent.yaml
+          volumeMounts:
+            - name: configs
+              mountPath: /etc/consent-cm
+              readOnly: true
+            - name: app-secrets
+              mountPath: /etc/service-account.json
+              subPath: service-account.json
+              readOnly: true
+            - name: consent-prometheusjmx-jar
+              mountPath: /etc/prometheusjmx/prometheusjmx.jar
+              subPath: prometheusjmx.jar
+          readinessProbe:
+            httpGet:
+              path: /status
+              port: 8080
+            initialDelaySeconds: 60
+            periodSeconds: 10
+            timeoutSeconds: 1
+            failureThreshold: 10
+            successThreshold: 1
+        - name: oidc-proxy
+          image: "{{ .Values.proxyImageRepository }}:{{ .Values.proxyImageVersion }}"
+          ports:
+            - containerPort: 443
+          volumeMounts:
+            - name: configs
+              mountPath: /etc/apache2/sites-available/site.conf
+              subPath: site.conf
+              readOnly: true
+            - name: proxy-secrets
+              mountPath: "/etc/ssl"
+              readOnly: true
+        - name: {{ .Chart.Name }}-sqlproxy
+          image: broadinstitute/cloudsqlproxy:1.11_20191120
+          envFrom:
+            - configMapRef:
+                name: configs
```

```suggestion
                name: consent-configmap
```
I think you are trying to reference the volume created from the config map on line 131.
This parameter expects the name of k8s config map rather than the volume I think.